### DISCUSSION TITLE
Facing issue when upgrading to 4.0

### DISCUSSION THREAD

**Discussion by kvinita** [clientIO/joint#2603] [Q&A] [answered]
### Introduction

My team recently purchased the JointJS+ 4.0 license. 
To integrate the license with the existing application, I replaced the build/package folder. After that, I deleted node modules & package-lock.json and ran `npm i`.

I replaced the existing import statements with `import joint from 'build/package/joint-plus';`

After that, I attempted to run `ng s`, which raised the below compile time errors:
![image](https://github.com/clientIO/joint/assets/124018900/b78b5789-0239-4ecc-8f88-7d417fabd147)

May I know what is causing this issue? Let me know if there are any more details required to troubleshoot this.

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kvinita** [comment]
The issue is irreproducible

**Reply by kumilingus** [reply, marked as answer]
You can try to find out how people generally deal with this exact error. For example: https://stackoverflow.com/a/51758340
